pyiron/lammps/control.py

```diff
@@ -701,10 +709,33 @@ def _measure_mean_value(self, key_pyiron, key_lmp, every):
                 freq (int): output frequency (default: n_print)
 
             Comments:
-                Currently available keys: 'energy_pot', 'energy_tot', 'temperature', 'volume'
-                Future keys: 'cells', 'forces', 'positions', 'pressures', 'unwrapped_positions', 'velocities'
+                Currently available keys: 'energy_pot', 'energy_tot', 'temperature', 'volume',
+                                          'pressures', 'positions', 'forces, 'velocities'
+                Future keys: 'cells'
         """
-        self['variable___{}'.format(key_pyiron)] = 'equal {}'.format(key_lmp)
-        self['fix___mean_{}'.format(key_pyiron)] = 'all ave/time '+str(every)+' ${mean_repeat_times} ${thermotime} v_'+str(key_pyiron)
-        self['thermo_style'] = self['thermo_style']+' f_mean_{}'.format(key_pyiron)
+        if isinstance(key_lmp, str):
+            self['variable___{}'.format(key_pyiron)] = 'equal {}'.format(key_lmp)
+            self['fix___mean_{}'.format(key_pyiron)] = 'all ave/time '+str(every)+' ${mean_repeat_times} ${thermotime} v_'+str(key_pyiron)
+            self['thermo_style'] = self['thermo_style']+' f_mean_{}'.format(key_pyiron)
+        else:
+            for ii, _ in enumerate(key_lmp):
```

Comment:
As a matter of style I'd prefer to have
```
else:
  if atom is True:
    for loop
    other stuff
  else:
    for loop
    other stuff
```

Because I find it easier to read. What's your opinion? If you prefer it as is, leave it.

Reply:
I don't have a preference, but it's true that by doing so I can spare a few lines. Thanks!